packages/smooth_app/lib/pages/preferences/country_selector/country_selector.dart

```diff
@@ -20,15 +17,14 @@ part 'country_selector_provider.dart';
 
 /// A button that will open a list of countries and save it in the preferences.
 class CountrySelector extends StatelessWidget {
-  const CountrySelector({
-    required this.forceCurrencyChange,
-    this.textStyle,
-    this.padding,
-    this.icon,
-    this.inkWellBorderRadius,
-    this.loadingHeight = 48.0,
-    this.autoValidate = true,
-  });
+  const CountrySelector(
+      {required this.forceCurrencyChange,
+      this.textStyle,
+      this.padding,
+      this.icon,
+      this.inkWellBorderRadius,
+      this.loadingHeight = 48.0,
+      this.autoValidate = true});
```

Comment:
```suggestion
  const CountrySelector({
    required this.forceCurrencyChange,
    this.textStyle,
    this.padding,
    this.icon,
    this.inkWellBorderRadius,
    this.loadingHeight = 48.0,
    this.autoValidate = true,
  });
```
Please don't change the format as:
1. it forces me to review a code that hasn't been changed
2. it's typically the way we format here, with a last comma